starname/chain.json

```diff
@@ -75,6 +75,10 @@
   },
   "apis": {
     "rpc": [
+      {
```

Comment:
Please move these additions to the bottom

Reply:
Done